apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -571,7 +568,34 @@ describe('ResearchOutputs controller', () => {
       ).rejects.toThrow(NotFoundError);
     });
 
-    test('Should update the research output and return it', async () => {
+    test('Should update a published existing research output and return it', async () => {
+      const researchOutputUpdateData: ResearchOutputUpdateData = {
+        ...getResearchOutputUpdateData(),
+      };
+      researchOutputDataProviderMock.update.mockResolvedValueOnce(
+        researchOutputId,
+      );
+
+      const result = await researchOutputs.update(
+        researchOutputId,
+        researchOutputUpdateData,
+        { publish: false },
+      );
+
+      const researchOutputUpdateDataObject =
+        getResearchOutputUpdateDataObject();
+
+      expect(result).toEqual(getResearchOutputResponse());
+      expect(researchOutputDataProviderMock.update).toBeCalledWith(
+        researchOutputId,
+        researchOutputUpdateDataObject,
+        { publish: false },
+      );
+    });
+
+    test('Should update a draft research output and return it', async () => {
+      const mockDate = new Date('2010-01-01');
+      const spy = jest.spyOn(global, 'Date').mockImplementation(() => mockDate);
```

Comment:
can't you just use jest mock timers method fake time? I realise this was probably copied from before but I think it would be a good time to look into a better alternative